py_experimenter/experimenter.py

```diff
@@ -29,58 +29,101 @@ def __init__(self,
                  database_name: str = None,
                  name='PyExperimenter'):
         """
-        Initialize PyExperimenter with the configuration file. If table_name or database_name are given they overwrite the
-        values in the configuration file.
-
-        :param config_file: Path to the configuration file. Defaults to 'config/configuration.cfg'.
-        :param table_name: Name of the table in the database. If None, the table name is taken from the configuration file. Defaults to None.
-        :param database_name: Name of the database. If None, the database name is taken from the configuration file. Defaults to None.
+        Initializes the PyExperimenter with the given information.
+
+        :param config_file: The path to the experiment configuration file. Defaults to 'config/configuration.cfg'.
+        :type config_file: str, optional
+        :param credential_path: The path to the database configuration file storing the credentials for the database connection, i.e., host, user and password. Defaults to 'config/database_credentials.cfg'.
+        :type credential_path: str, optional
+        :param table_name: The name of the database table. If None, the table table name is taken from the experiment configuration file. Defaults to None.
```

Comment:
Maybe make more explicit, that this overwrites the config file if there is also a name given there.

Reply:
Same for the settings below.